### ISSUE TITLE
Nice work, but in your view model class the view model inject annotation is deprecated

### ISSUE THREAD

**Issue by hithamalbasheir** [MohAbbasher/ArchitectureSimple#1]
You should use @hiltviewmodel with @inject, https://stackoverflow.com/questions/66185820/dagger-hilt-assisted-and-viewmodelinject-is-deprecated-in-dagger-hilt-view

**Comment by MohAbbasher** [comment]
yes now `@Inject` for `viewModel` is deprecated .fell free to fork and update it ,then you can make PR I will be happy to read your code.
